perun-openapi/openapi.yml

```diff
@@ -4737,6 +4737,296 @@ paths:
+        '200':
+          $ref: '#/components/responses/GroupResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: CreateGroup
+              description: "input to createGroup"
+              type: object
+              required:
+                - parentGroup
+                - name
+                - description
+              properties:
+                parentGroup: {type: integer, description: "parentGroup id", nullable: false}
+                name: {type: string}
+                description: {type: string}
+
+  /json/groupsManager/deleteGroups:
+    post:
+      tags:
+        - GroupsManager
+      operationId: deleteGroups
+      summary: Forcefully deletes a list of groups (remove all group members, remove group from resources).
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: DeleteGroups
+              description: "input to deleteGroups"
+              type: object
+              required:
+                - groups
+                - forceDelete
+              properties:
+                groups:
+                  type: array
+                  items:
+                    type: integer
+                forceDelete: {type: boolean}
+
+  /json/groupsManager/moveGroup/dg-mg:
+    post:
+      tags:
+        - GroupsManager
+      operationId: moveGroupWithDestinationGroupMovingGroup
+      summary: Moves "movingGroup" (including subGroups) under "destinationGroup" as subGroup within same Vo. Indirect group members are also processed during move operation.
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: moveGroup
+              description: "input to moveGroup"
+              type: object
+              required:
+                - destinationGroup
+                - movingGroup
+              properties:
+                destinationGroup: {type: integer}
+                movingGroup: {type: integer}
+
+  /json/groupsManager/addMembers:
+    post:
+      tags:
+        - GroupsManager
+      operationId: addMembers
+      summary: Adds members to a group. If already a member of the group, the member will be skipped. Non-empty list of members expected, if empty, no member will be added.
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
```

Comment:
please replace the request body with URL parameter of names "group" and "members[]"